src/components/ArticleNavigations/ArticleNavigations.scss

```diff
@@ -0,0 +1,28 @@
+@use '~@gravity-ui/page-constructor/styles/variables.scss' as pcVariables;
+@use '../../variables.scss';
+
+$block: '.#{variables.$ns}article-navigations';
+
+#{$block} {
+    display: flex;
+    gap: 24px;
```

Comment:
lets use [Module and spacing system](https://gravity-ui.com/design/branding/module) at the page that describe it

24px = --g-spacing-6